src/clib/lib/job_queue/torque_driver.cpp

```diff
@@ -289,6 +303,22 @@ torque_job_type *torque_job_alloc() {
     return job;
 }
 
+std::string build_resource_string(int num_nodes, std::string cluster_label,
+                                  int num_cpus_per_node,
+                                  std::string memory_per_job) {
+    std::string resource_string = "nodes=" + std::to_string(num_nodes);
+
+    if (!cluster_label.empty())
+        resource_string.append(":" + cluster_label);
+
+    resource_string.append(":ppn=" + std::to_string(num_cpus_per_node));
+
+    if (!memory_per_job.empty())
```

Comment:
Right, I thought that this line: `torque_driver_set_option(torque_driver, TORQUE_MEMORY_PER_JOB, "1g");`
makes the default to 1GB , so I guess this is not the case.